manifest.php

```diff
@@ -34,11 +34,11 @@
 	'label' => 'Open-web item model',
     'description' => 'Open Web Item model allows users to create items with rich HTML content',
     'license' => 'GPL-2.0',
-    'version' => '3.0.0',
+    'version' => '3.1.0',
 	'author' => 'Open Assessment Technologies',
 	'requires' => array(
         'taoItems' => '>=2.19.0',
```

Comment:
version should be 2.22.0

Reply:
also check, please tao dependency for all PR's

Reply:
But I think that you still need to increase version of the taoItems because of `taoItems_models_classes_ItemsService::PROPERTY_ITEM_MODEL`

Reply:
@ionutpad

Reply:
Yes you're right I don't have time to do this for all PR's, so I will do it as soon as I have time.
Thanks for the hint